### DISCUSSION TITLE
v2.0.35b2

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#348] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.35b2'>v2.0.35b2</a>.</em>

**Comment by harphere** [comment]
I'm getting some pretty big distance numbers for my iBeacon via HA companion app:
`  "distance": 1444935.68,
  "speed": 262.29,
  "rssi": -58,
  "raw": 1389495.2,
  "distance": 1435312.96,
`
My phone is laying on top of the ESP32.

**Reply by DTTerastar** [reply]
What's your rss@1m?

**Reply by harphere** [reply]
I don't fully understand how to interpret this but my device at 1m from the ESP32 with a 0 max distance configured yields:
```
{
  "id": "iBeacon:xxx-100-1",
  "idType": 115,
  "rssi@1m": 157,
  "rssi": -68,
  "raw": 2682694.56,
  "distance": 2344392.48,
  "speed": 24364.29,
  "distance": 2369184,
  "speed": 22562.73,
  "mac": "yyy"
}
```

**Reply by DTTerastar** [reply]
rssi@1m is the rssi we'd expect if the device was exactly 1 meter away.  Since that value is WILDLY wrong that's what feeds the terrible distance.  It's definitely a bug!  I'll fix it. Thanks for testing!

**Reply by DTTerastar** [reply]
The latest prerelease contains a fix, please test!  (It should update to it automatically)
